canonical-web-and-design/cn.ubuntu.com
# Convert app from Django to Flask

## Done

Convert cn.ubuntu.com from Django to Flask!!!

Without realising I removed the cloudinary. But we should add the image_template module.

## QA

- `./run`
- 127.0.0.1:8010
- Make sure to navigate through all pages (/blog /internet-of-things ...) make sure that they still display something.
- I basically run the linkchecker to make sure that no pages were 500 in there and it seems fine for now.
